ngsa/src/main/resources/application.properties

```diff
@@ -37,13 +37,14 @@ spring.banner.location=classpath:static/startup-banner.txt
 server.shutdown=graceful
 spring.lifecycle.timeout-per-shutdown-phase=10s
 
-burst-service=ngsa-java
-burst-target=60
-burst-max=80
+burst-service=${BURST_SERVICE:ngsa-java}
+burst-target=${BURST_TARGET:60}
+burst-max=${BURST_MAX:80}
+burst-header=${BURST_HEADER:false}
```

Comment:
What's the purpose of `BURST_SERVICE` here, does it mean to use that env variable if present? I don't think that's needed because you can use `BURST_SERVICE` as is to set the property here.

Reply:
`BURST_SERVICE` is actually the value of `--burst-service` arg, so if env var is set, it will take precedence over default value of `ngsa-java`

Reply:
@sivamu did you mean there isn't a need to override service name?